controllers/release/release_adapter_test.go

```diff
@@ -70,7 +70,14 @@ var _ = Describe("Release Adapter", Ordered, func() {
 			},
 			Spec: ecapiv1alpha1.EnterpriseContractPolicySpec{
 				Description: "test-policy-description",
-				Sources:     []string{"foo"},
+				Sources: []ecapiv1alpha1.PolicySource{
```

Comment:
This should just be `Source` not `PolicySource` based on how they have things now